cmd/explored/main.go

```diff
@@ -298,7 +298,8 @@ func runRootCmd(ctx context.Context, log *zap.Logger) error {
 	}
 	go ex.Start(ctx)
 
-	api := api.NewServer(e, cm, s, ex)
+	manualScanPassword := os.Getenv("MANUAL_SCAN_PASSWORD")
```

Comment:
we already have `EXPLORED_CONFIG_FILE`, so should we use `EXPLORED_HOST_SCAN_PASSWORD` or something that mentions `AUTH_` perhaps?

Reply:
I’d stay consistent with the other apps and go `EXPLORED_API_PASSWORD`